emulated_hue/api.py

```diff
@@ -21,11 +21,9 @@
 
 LOGGER = logging.getLogger(__name__)
 
-DESCRIPTION_FILE = os.path.join(
-    os.path.dirname(os.path.abspath(__file__)), "description.xml"
-)
-
-CLIP_FILE = os.path.join(os.path.dirname(os.path.abspath(__file__)), "clip.html")
+STATIC_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "web_static")
+DESCRIPTION_FILE = os.path.join(STATIC_DIR, "description.xml")
+CLIP_FILE = os.path.join(STATIC_DIR, "clip.html")
```

Comment:
We need to move clip to /debug/clip.html to be in line with the actual bridge.

Reply:
I'm not sure if a subfolder works in the static content hosting. Otherwise we'll have to revert it.
My mistake I thought it was clip.html at the root. Did not look very well I suppose.

Reply:
Maybe just add another route at line 132